src/js/controller/instream-html5.js

```diff
@@ -145,6 +145,10 @@ define([
                 provider.mute(_model.get('mute'));
 
                 _adModel.on('change:state', changeStateEvent, _this);
+            } else if(provider.state === 'playing') {
```

Comment:
The state is still `playing` on mobile because the html5 provider does not change the state to `loading` on mobile devices:
```
if (!_isMobile) {
                // don't change state on mobile because a touch event may be required to start playback
                _this.setState(states.LOADING);
            }
```
In the case of ads, a touch event is not required to start playback of the next ad in an ad pod, so it's safe to assume the provider is in a `loading` state when `_checkProvider` is called in the `load` function in instream-html5.js. 

An alternative would be to set the provider's state to something like `skipped` or `stopped` in instream-adapter.js when an ad is skipped, but I did not want to introduce a new state to the provider if I could avoid it. The current states the provider cycles through for ad playback are:
* idle
* loading
* playing
* complete

When an ad plays to completion and the next ad starts, the provider goes through the following state changes: `playing > complete > loading > playing`. For skipped ads, it's stuck on `playing` even though we've stopped playback and started loading the next ad. The model's state - set to `buffering` between each ad in an ad pod - receives no notification that the new ad is loaded and has started playing, so the buffering icon displays until the provider's state changes to `complete` or the player switches from the instream provider back to the main provider that resumes normal media playback.